task-sdk/tests/task_sdk/definitions/decorators/test_task_group.py

```diff
@@ -174,14 +175,10 @@ def tg(a, b):
 
         tg.partial(a=1).expand(b=t()["values"])
 
-    with pytest.raises(ValueError) as ctx:
+    expected_msg = "cannot map over XCom with custom key 'values' from <Task(_PythonDecoratedOperator): t>"
+    with pytest.raises(ValueError, match=re.escape(expected_msg)):
```

Comment:
Nitpick (here and below), feel free to close or ignore this, but I'm pretty sure you can define the match string as a raw-string like this and save the import, but I haven't tested this.

```suggestion
    expected_msg = r"cannot map over XCom with custom key 'values' from <Task(_PythonDecoratedOperator): t>"
    with pytest.raises(ValueError, match=expected_msg):
```